sql/start.sql

```diff
@@ -0,0 +1,128 @@
+CREATE DATABASE movies_db;
+
+-- Actors table
+CREATE TABLE public.actors
+(
+    id serial NOT NULL,
+    name character varying(45) NOT NULL,
+    birth_year integer NOT NULL,
+    PRIMARY KEY (id),
+    UNIQUE (name)
+);
+
+ALTER TABLE public.actors
+    OWNER to postgres;
+
+-- Genres table
+CREATE TABLE public.genres
+(
+    id serial NOT NULL,
+    name character varying(45) NOT NULL,
+    PRIMARY KEY (id),
+    UNIQUE (name)
+);
+
+ALTER TABLE public.genres
+    OWNER to postgres;
+
+-- Movies table
+CREATE TABLE public.movie
+(
+    id integer NOT NULL DEFAULT nextval('movie_id_seq'::regclass),
+    title character varying(70) COLLATE pg_catalog."default" NOT NULL,
+    description character varying(140) COLLATE pg_catalog."default" NOT NULL,
```

Comment:
1. Are you sure that 70 and 140 chars will be enough for the title and description?
2. Do you really need collation for these fields?